### DISCUSSION TITLE
sys.print_exception error

### DISCUSSION THREAD

**Discussion by MikeGarber** [microsoft/pyright#7909] [General]
"print_exception" is not a known attribute of module "sys"Pylance[reportAttributeAccessIssue](https://github.com/microsoft/pyright/blob/main/docs/configuration.md#reportAttributeAccessIssue)
(function) print_exception: Unknown"

I patched around it by finding the json file (Actually called "pyrightconfig.schema.json" as opposed to “pyrightconfig.json” as described in your doc, but that's probably because Im using VSCode/Pylance) and turning reportAttributeAccessIssue to "warning".
But something's not quite kosher.

Works fine in Thonny (which I refuse to go back to).
Wasnt sure if this goes into "discussion" or "issues"

**Comment by erictraut** [comment]
I'm not sure what you mean. Are you saying that you think that the stdlib `sys` module has a symbol named `print_exception`? That doesn't appear to be the case, at least not in standard Python distributions. Perhaps you're using a non-standard version of Python line micropython?

In any case, pyright has no specialized knowledge of symbols in specific stdlib libraries. Like other Python type checkers, it relies on type stubs from the [typeshed project](https://github.com/python/typeshed). If you think that there is a missing symbol in the [sys/__init__.pyi](https://github.com/python/typeshed/blob/main/stdlib/sys/__init__.pyi) type stub, please report the issue to the maintainers of typeshed.

**Reply by MikeGarber** [reply]
I beginning to think this tool chain was never intended for the RPi Nano (rp20400).
An I right?
For VS Code, MicroPico seems to take it all into account (despite calling for Pylance as a prereq)

Sorry for your trouble.
